From: Simon McVittie <smcv@debian.org>
Date: Thu, 18 Jul 2019 09:20:57 +0100
Subject: gtk4.pc: Move third-party libraries from Libs to Libs.private

This avoids "overlinking". These libraries are an implementation detail
of particular backends, so they only need to appear on library consumers'
linker command-lines if the dependency cannot be picked up from the
shared library automatically (when linking statically, or when building
on a deficient OS that doesn't support transitive dependencies between
shared libraries, in which case pkg-config should be built with
--enable-indirect-deps).

Signed-off-by: Simon McVittie <smcv@debian.org>
---
 gtk4.pc.in | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/gtk4.pc.in b/gtk4.pc.in
index e721361..b6822bd 100644
--- a/gtk4.pc.in
+++ b/gtk4.pc.in
@@ -12,5 +12,6 @@ Description: GTK Graphical UI Library
 Version: @VERSION@
 Requires: @GDK_PACKAGES@ @GSK_PACKAGES@ @GTK_PACKAGES@
 Requires.private: @GDK_PRIVATE_PACKAGES@ @GSK_PRIVATE_PACKAGES@ @GTK_PRIVATE_PACKAGES@
-Libs: -L${libdir} -lgtk-4 @GDK_EXTRA_LIBS@ @GSK_EXTRA_LIBS@ @GTK_EXTRA_LIBS@
+Libs: -L${libdir} -lgtk-4
+Libs.private: @GDK_EXTRA_LIBS@ @GSK_EXTRA_LIBS@ @GTK_EXTRA_LIBS@
 Cflags: -I${includedir}/gtk-@GTK_API_VERSION@ @GDK_EXTRA_CFLAGS@ @GSK_EXTRA_CFLAGS@ @GTK_EXTRA_CFLAGS@
